### PR TITLE
i18n: import

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: lighthouse-core/lib/i18n/locales/ar.json [hunk 1]
```diff
@@ -2060,6 +2069,9 @@
   "lighthouse-core/report/html/renderer/util.js | runtimeUnknown": {
     "message": "غير معروف"
   },
+  "lighthouse-core/report/html/renderer/util.js | showRelevantAudits": {
+    "message": "Show audits relevant to:"
```
Comment (connorjclark) [inline lighthouse-core/lib/i18n/locales/ar.json:2073]: uhhhh?

File: lighthouse-core/lib/i18n/locales/ca.json [hunk 2]
```diff
@@ -2060,6 +2069,9 @@
   "lighthouse-core/report/html/renderer/util.js | runtimeUnknown": {
     "message": "Desconegut"
   },
+  "lighthouse-core/report/html/renderer/util.js | showRelevantAudits": {
+    "message": "Show audits relevant to:"
```
Comment (connorjclark) [inline lighthouse-core/lib/i18n/locales/ca.json:2073]: same...

prob tons more, should we have a test that the non-en strs don't === the en strs?

File: lighthouse-core/lib/i18n/locales/ar.json (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/lib/i18n/locales/ar.json:2073]: doesn't look good but I think that setup with that string is already rough for rtl i18n (anything ending with a colon and saying "this stuff here"), maybe that's why it was skipped?

Comment (connorjclark) [reply lighthouse-core/lib/i18n/locales/ar.json:2073]: Don't think it was skipped for that reason, or at least others weren't (ca/Catalan is not rtl).

File: lighthouse-core/lib/i18n/locales/de.json [hunk 3]
```diff
@@ -2087,6 +2099,30 @@
   "lighthouse-core/report/html/renderer/util.js | warningHeader": {
     "message": "Warnungen: "
   },
+  "lighthouse-treemap/app/src/util.js | allLabel": {
+    "message": "All"
```
Comment (patrickhulce) [inline lighthouse-core/lib/i18n/locales/de.json:2103]: seems like all of the utils just weren't translated

**Review by patrickhulce — APPROVED** [review]
behavior is the same with the english ones, so doesn't seem to hurt anything at least.

File: lighthouse-core/lib/i18n/locales/ar.json (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/lib/i18n/locales/ar.json:2073]: yeah I found all the util.js german ones were just skipped.

**Comment by brendankenny** [comment]
all strings are filled out by the tc dump, but it uses `en` strings if a translation isn't available yet

**Pull request merged**